resources/nl-core/nl-core-Vaccination-event.xml

```diff
@@ -60,9 +65,45 @@
       <definition value="The pharmaceutical product of the administered vaccination." />
       <comment value="If this reference to zib PharmaceuticalProduct is used, the MedicationCode (`Medication.code`) in the referenced resource SHALL be the same as `Immunization.vaccineCode` of the current resource." />
     </element>
+    <element id="Immunization.extension:anatomicalLocation">
+      <path value="Immunization.extension" />
+      <sliceName value="anatomicalLocation" />
+      <min value="0" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-Vaccination.AnatomicalLocation" />
+      </type>
+      <isModifier value="false" />
+    </element>
+    <element id="Immunization.extension:anatomicalLocation.value[x]">
+      <path value="Immunization.extension.value[x]" />
+      <definition value="The anatomical location of where the vaccination is administered." />
+    </element>

```

Comment:
https://hl7.org/fhir/R4/immunization-definitions.html#Immunization.site is een core element wat hiervoor bedoeld lijkt

Reply:
@jduwel Immunization.site zou niet voldoende om alle code(lijsten) van anatomicalLocation te ondersteunen had ik begrepen van @pieter-edelman-nictiz.

Reply:
@jduwel The anatomicalLocation extension is removed and we added the anatomicalLocation as a profile to codable concept .site. This reflects how anatomicalLocation is represented in nl core Problem.